.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: node_js
-cache: yarn
+cache:  yarn
 node_js:
 - "8"
 install:
-- yarn install
-script: gitbook install && gitbook build
+  - yarn install
+  - yarn global add markdownlint-cli
+script:
+  - gitbook install && gitbook build
+  - markdownlint . --ignore node_modules
```

Comment:
Should we run the linter before building? Usually linting is quicker which would cause the CI to break faster.

Reply:
Good idea, updated!